### ISSUE TITLE
Choose the alternative audio loading method.

### ISSUE THREAD

**Issue by Katsugami** [clementinise/CustomSounds#11]
Hello,

Would it be possible to have the option, in the configuration file, to specify the loading order of different songs based on a plugin?

To elaborate further, I use two mods that modify several songs, but there are three songs that both mods modify. However, when we launch the game, it loads the audio and replaces them in alphabetical order of the mods.

The issue is that for these three particular songs, I would like the mod with the folder starting with the letter N to be loaded first, rather than the one starting with the letter A.

This would allow that when I share my profile configuration with R2ModMan, everyone has the same songs without needing them to open the mods, swap, or delete the audios that will be loaded at startup.

In summary, from a configuration file, I would like to be able to add the location links of plugins that should be loaded first, and then the rest, or specify one that should be loaded first followed by another in second place.

Thank you sincerely.
